### ISSUE TITLE
well done, hymn!

### ISSUE THREAD

**Issue by nagexiucai** [zotroneneis/machine_learning_basics#7]
very good.

in chapter "linear-regression":

"n_iters" is omitted in section "training with gradient descent";

"c='orange'" is omitted in section "visualize test predictions";

in chapter "logistic-regression":

"* 100" is omitted in section "testing the model";

in chapter "k-nearest-neighbor":

"regregression" may be typo at the start;

"plt.show()" is omitted in section "dataset";

"test accuracy with k = 8" and "*8-like" characters may be typos, must be 4;

in chapter "simple neural net":

"\textit{sigmoid}" may be typo in section "forward pass";

in chapter "softmax regression":

"one-hot encoded" in section "step iii", confused by "So $y_k^{(i)}$ is $1$ is the target class for $\boldsymbol{x}^{(i)}$ is k, otherwise $y_k^{(i)}$ is $0$.", what is the meaning of "is 1 is the target...";

continuous updating...

**Comment by zotroneneis** [comment]
Thanks a lot, I have corrected the points!
